ejb3/src/main/java/org/jboss/as/ejb3/component/interceptors/LoggingInterceptor.java

```diff
@@ -64,6 +65,9 @@ public Object processInvocation(InterceptorContext interceptorContext) throws Ex
         try {
             // we just pass on the control and do our work only when an exception occurs
             return interceptorContext.proceed();
+        } catch ( EJBComponentUnavailableException ex) {
+            EjbLogger.EJB3_INVOCATION_LOGGER.warn(ex);
```

Comment:
@bstansberry sounds indeed like a good enhancement to the fix. I'll work on that soon and push an other PR.